modules/gis/docs/api-reference/geojson-to-binary.md

```diff
@@ -19,51 +19,92 @@ const geoJSONfeatures = await load(url, MVTLoader, loaderOptions);
 * Default options are:
 *
 * {
+*   coordLength: derived from data
+*   numericPropKeys: derived from data
 *   PositionDataType: Float32Array
 * }
 */
+const options = {PositionDataType: Float32Array};
 const binaryArrays = geojsonToBinary(geoJSONfeatures, options);
 ```
 
 ## Outputs
 
-### TypedArrays
-
 `geojsonToBinary` returns an object containing typed arrays sorted by geometry
-type. `positions` corresponds to 2D or 3D coordinates; `objectIds` returns the
-index of the vertex in the initial `features` array.
+type. `positions` is a flat array of coordinates; `globalFeatureIds` references
+indices in the original `features` array; `featureIds` references feature
+indices of the same geometry type; `numericProps` contains `TypedArray`s
+generated from numeric feature properties; `properties` is an array of
+non-numeric property objects of the given geometry type.
+
+Each `TypedArray` is wrapped inside an _accessor object_, where `.value` contains the raw numeric data, and `.size` gives the number of values per vertex. For example,
+
+```js
+positions: {value: Float32Array, size: 3}
+```
+
+corresponds to 3D coordinates, where each vertex is defined by three numbers.
 
 ```js
 {
   points: {
     // Array of x, y or x, y, z positions
-    positions: {value: Float32Array, size: coordLength},
+    positions: {value: PositionDataType, size: coordLength},
     // Array of original feature indexes by vertex
-    objectIds: {value: Uint32Array, size: 1},
+    globalFeatureIds: {value: Uint16Array || Uint32Array, size: 1},
+    // Array of Point feature indexes by vertex
+    featureIds: {value: Uint16Array || Uint32Array, size: 1},
+    // Object with accessor objects for numeric properties
+    // Numeric properties are sized to have one value per vertex
+    numericProps: {
+        numericProperty1: {value: Float32Array, size: 1}
+    }
+    // Array of objects with non-numeric properties from Point geometries
+    properties: [{PointFeatureProperties}]
   },
   lines: {
-    // Indices within positions of the start of each individual LineString
-    pathIndices: {value: Uint32Array, size: 1},
     // Array of x, y or x, y, z positions
-    positions: {value: Float32Array, size: coordLength},
+    positions: {value: PositionDataType, size: coordLength},
```

Comment:
We are in the process of introducing typescript defintions for loaders.gl 2.2. With minimal extra effort, we could actually turn what you wrote here into typescript types in a `geojson-to-binary.d.ts`. Look for other `.d.ts` files to see how it is done.